app/js/directives.js

```diff
@@ -1155,16 +1150,16 @@ angular.module('myApp.directives', ['myApp.filters'])
       }
 
       var transform = false
-      var trs = ['transform', 'webkitTransform', 'MozTransform', 'msTransform', 'OTransform'],
-        i
+      var trs = ['transform', 'webkitTransform', 'MozTransform', 'msTransform', 'OTransform']
+      var i
       for (i = 0; i < trs.length; i++) {
         if (trs[i] in historyMessagesEl.style) {
           transform = trs[i]
           break
         }
       }
 
-      var animated = transform && false ? true : false
+      var animated = transform && false ? true : false // ?
```

Comment:
@bartist I [guess](https://github.com/zhukov/webogram/blob/7dcc2df4d4968a657b7e3f69cc23977d8145060f/app/js/directives.js#L610) it should be `transform ? true : false` (I'd prefer `!!trasform`)